Mathlib/RingTheory/WittVector/InitTail.lean

```diff
@@ -103,11 +103,9 @@ theorem select_add_select_not : ∀ x : 𝕎 R, select P x + select (fun i => ¬
     AlgHom.map_mul, bind₁_X_right, bind₁_C_right, ← Finset.sum_add_distrib, ← mul_add]
   apply Finset.sum_congr rfl
   refine' fun m _ => mul_eq_mul_left_iff.mpr (Or.inl _)
-  rw [ite_pow, zero_pow (pow_pos hp.out.pos _)]
+  rw [ite_pow, zero_pow (pow_ne_zero _ hp.out.ne_zero)]
```

Comment:
This idiom was repeated a few times in the diff, but I think this is *slightly* nicer (untested):
```suggestion
  rw [ite_pow, zero_pow (pow_pos hp.out.pos _).ne']
```
It's possible I'm missing something though.

Reply:
I understand the idea, but I'm not sure this is better. The "correct" proof of course is
```suggestion
  rw [ite_pow, zero_pow (by positivity)]
```
but I haven't made it work yet (nor am I sure I can make it work without a performance penalty, actually). I am happy to wait for this to be figured out before merging this PR if you think that's important.